Formula/flagd.rb

```diff
@@ -7,6 +7,11 @@ class Flagd < Formula
   license "Apache-2.0"
   head "https://github.com/open-feature/flagd.git", branch: "main"
 
+  livecheck do
```

Comment:
I wonder if an explicit `livecheck` block is necessary here. By default, `livecheck` is able to retrieve the latest version correctly, and it doesn't seem like there are any extraneous tags we'd like to avoid.
```
Formula:          flagd
Livecheckable?:   No

URL:              https://github.com/open-feature/flagd.git
Strategies:       Git
Strategy:         Git

Matched Versions:
0.0.1 => #<Version:0x00007fc5c8069ec0 @version="0.0.1", @detected_from_url=false>
0.0.2 => #<Version:0x00007fc5c8069e70 @version="0.0.2", @detected_from_url=false>
0.0.2-1 => #<Version:0x00007fc5c8069c90 @version="0.0.2-1", @detected_from_url=false>
0.0.3 => #<Version:0x00007fc5c8069ba0 @version="0.0.3", @detected_from_url=false>
0.0.4 => #<Version:0x00007fc5c8069998 @version="0.0.4", @detected_from_url=false>
0.0.5 => #<Version:0x00007fc5c8069858 @version="0.0.5", @detected_from_url=false>
0.0.6 => #<Version:0x00007fc5c8069678 @version="0.0.6", @detected_from_url=false>
0.0.7 => #<Version:0x00007fc5c8069358 @version="0.0.7", @detected_from_url=false>
0.0.8 => #<Version:0x00007fc5c80691c8 @version="0.0.8", @detected_from_url=false>
0.0.9 => #<Version:0x00007fc5c8068e80 @version="0.0.9", @detected_from_url=false>
0.1.0 => #<Version:0x00007fc5c8068db8 @version="0.1.0", @detected_from_url=false>
0.1.1 => #<Version:0x00007fc5c8068980 @version="0.1.1", @detected_from_url=false>
0.2.0 => #<Version:0x00007fc5c80681b0 @version="0.2.0", @detected_from_url=false>
0.2.1 => #<Version:0x00007fc5c8073df8 @version="0.2.1", @detected_from_url=false>
0.2.2 => #<Version:0x00007fc5c8072e30 @version="0.2.2", @detected_from_url=false>
0.2.3 => #<Version:0x00007fc5c8071eb8 @version="0.2.3", @detected_from_url=false>
0.2.4 => #<Version:0x00007fc5c8070ba8 @version="0.2.4", @detected_from_url=false>
0.2.5 => #<Version:0x00007fc5c8070a68 @version="0.2.5", @detected_from_url=false>
0.2.7 => #<Version:0x00007fc5c8070838 @version="0.2.7", @detected_from_url=false>
0.3.0 => #<Version:0x00007fc5c80707c0 @version="0.3.0", @detected_from_url=false>
0.3.1 => #<Version:0x00007fc5cbc57ce8 @version="0.3.1", @detected_from_url=false>

flagd (guessed): 0.3.1 ==> 0.3.1
```